hugo.yaml

```diff
@@ -38,8 +41,17 @@ params:
       [CC BY 4.0](https://creativecommons.org/licenses/by/4.0) |
     from_year: 2023
       #privacy_policy: link to privacy policy
-  version_menu: Releases
+
+  version_menu: Documentation 
+  version_menu_pagelinks: false
   url_latest_version: https://parodos.dev
+  versions:
+    - version: main
+      url: '/main/docs'
+    - version: v1.1.x
+      url: '/v1.1.x/docs'
+  version: main
```

Comment:
For now,  if this is the default selected version, I'd go with `v1.1.x` which is the latest stable version available.